src/components/add-site.tsx

```diff
@@ -135,7 +135,12 @@ export default function AddSite( { className }: AddSiteProps ) {
 					</SiteForm>
 				</Modal>
 			) }
-			<Button variant="outlined" className={ className } onClick={ openModal }>
+			<Button
+				variant="outlined"
+				className={ className }
+				onClick={ openModal }
+				disabled={ isSiteAdding }
```

Comment:
I tried using `isBusy` here but it was not good looking in terms of design and also still opened the modal so I prefer to use `disabled`